internal/cmd/skupper/site/nonkube/site_delete.go

```diff
@@ -85,6 +91,28 @@ func (cmd *CmdSiteDelete) Run() error {
 	if err != nil {
 		return err
 	}
+	// if delete all, also remove all the other resources
+	if cmd.all {
```

Comment:
What if there are AccessTokens or Links to other sites?
Would it be fair to remove all resources from the input/resources instead?